src/anbox/qemu/pipe_connection_creator.h

```diff
@@ -55,6 +56,7 @@ class PipeConnectionCreator
     qemud_gsm,
     qemud_adb,
     bootanimation,
+    qemud_gps,
```

Comment:
Can we put it in order with the other `qemud_*` ones?

Reply:
Sure, can be done.